webapp/channels/src/components/purchase_modal/icon_message.tsx

```diff
@@ -3,28 +3,19 @@
 
 import classNames from 'classnames';
 import React from 'react';
-import {FormattedMessage} from 'react-intl';
 
 import './icon_message.scss';
-import ExternalLink from 'components/external_link';
 
 type Props = {
     icon: JSX.Element;
-    title?: string;
```

Comment:
It looks like this component was originally written to take strings for all these props and then the "formatted" versions were added. I got rid of those to simplify this